src/error.rs

```diff
@@ -253,17 +253,17 @@ impl<'src> ColorDisplay for Error<'src> {
           match io_error.kind() {
             io::ErrorKind::NotFound => write!(
               f,
-              "Backtick could not be run because just could not find `sh`:\n{}",
+              "Backtick could not be run because just could not find the shell command:\n{}",
```

Comment:
I think maybe just "the shell" is better.

```suggestion
              "Backtick could not be run because just could not find the shell:\n{}",
```